CHANGELOG.unreleased.md

```diff
@@ -13,9 +13,13 @@
 
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
+- [OVA import] Fix memory hogging behavior when the destination SR is slower than the file upload. (PR [#4920](https://github.com/vatesfr/xen-orchestra/pull/4920))
+
 ### Released packages
 
 > Packages will be released in the order they are here, therefore, they should
 > be listed by inverse order of dependency.
 >
 > Rule of thumb: add packages on top.
+
+xo-vmdk-to-vhd patch
```

Comment:
Are there other packages we should release?

Reply:
xo-server, but I guess it's always released, isn't it?

Reply:
`xo-web` and `xo-server` are indeed special, and we have not decided exactly what should be the workflow for them.

Maybe we should add them normally until we decide and document otherwise?